packages/circuit-ui/components/ListItemGroup/ListItemGroup.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ReactElement, forwardRef, Ref, HTMLProps, ReactNode } from 'react';
+import { css } from '@emotion/core';
+
+import styled, { StyleProps } from '../../styles/styled';
+import { ReturnType } from '../../types/return-type';
+import { ListItemProps } from '../ListItem';
+
+interface BaseProps {
+  /**
+   * Children of type ListItem.
+   */
+  children: ReactElement<ListItemProps>[] | ReactElement<ListItemProps>;
```

Comment:
Not sure if list items should actually be react elements or if they should be objects, and we just build the elements under the hood. We've been going for the second option lately for some components (new wip ButtonGroup, new wip notifications...) but mostly because the wrapper had to handle logic for the children. In this case children might make more sense though. cc @connor-baer other. thoughts?

Reply:
Do you mean that instead of `children: ReactElement<ListItemProps>[]` we'll have something like `items: ListItemProps[]` and we'll handle the `map`-ing and props spreading internally. I'm generally not very happy with seeing props as objects but it would make the styling of the `ListItem` component in `ListItemGroup` somewhat easier and cleaner. If that's the direction we're heading with the other components I'm happy to change the `ListItemGroup`.